### DISCUSSION TITLE
graveyarded beatmaps showing up as pending on "play beatmap" screen

### DISCUSSION THREAD

**Discussion by Ronikusu** [ppy/osu#23159] [Q&A] [answered]
The beatmap status is not being updated on the "play beatmap" screen. 

![image](https://user-images.githubusercontent.com/124932615/230700048-a46c8daf-8cd3-4139-b1fa-507c9409ac05.png)

What the details screen says:

![image](https://user-images.githubusercontent.com/124932615/230700063-e67d5565-f5c0-4536-9f00-ae12194c2e28.png)

This also applies to revived beatmaps. It does not happen with newly qualified and ranked beatmaps becoming to loved.

**Comment by peppy** [comment]
Right now there's no flow to handle beatmaps entering a graveyarded state. This will likely need to be considered across at `osu-server-spectator`:

https://github.com/ppy/osu-server-spectator/blob/08bfe2a2d82261da53ddd16b5a411d6d775149d9/osu.Server.Spectator/Database/DatabaseAccess.cs#L271-L289

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu-server-spectator/issues/165
